cmd/localCI/pullRequest.go

```diff
@@ -75,11 +75,13 @@ type PullRequest struct {
 // canBeTested returns an error if the pull request cannot be tested
 func (pr *PullRequest) canBeTested() error {
 	if !pr.Mergeable {
+		ciLog.Debugf("the pull request is not mergeable")
 		return fmt.Errorf("the pull request is not mergeable")
```

Comment:
@amshinde ah, indeed, that is a bit neater.
I would rewrite, but @devimc is about to rework the code anyhow I believe - @devimc so, I presume you are OK to merge as-is as it will then change anyhow?